src/main/java/io/github/hildi/can/service/JavaSerializationService.java

```diff
@@ -0,0 +1,239 @@
+
+    private void setUserCreatedData(Date createdAt) {
+        user.setCreatedAt(createdAt);
+    }
+
+    private void getUserPropertiesFromFile(Scanner scanner) {
+        String[] line = scanner.nextLine().split(" = ");
+        String param = line[0].toLowerCase().trim();
+
+        getUserId(param, line);
+        getUserNickName(param, line);
+        getUserFirstName(param, line);
+        getUserLastName(param, line);
+        getUserEmail(param, line);
+        getUserPermissions(param, line);
+        getUserAttributes(param, line);
+        getUserCreatedData(param, line);
+    }
+
+    private void getUserId(String param, String[] line) {
+        if (param.equals("id".toLowerCase().trim())) {
+            userId = (long) Integer.parseInt(line[1]);
+        }
+    }
+
+    private void getUserNickName(String param, String[] line) {
+        if (param.equals("nickName".toLowerCase().trim())) {
+            userNickName = line[1];
+        }
+    }
+
+    private void getUserFirstName(String param, String[] line) {
+        if (param.equals("firstName".toLowerCase().trim())) {
+            firstName = line[1].trim();
+        }
+    }
+
+    private void getUserLastName(String param, String[] line) {
+        if (param.equals("lastName".toLowerCase().trim())) {
+            lastName = line[1].trim();
+        }
+    }
+
+    private void getUserEmail(String param, String[] line) {
+        if (param.equals("email".toLowerCase().trim())) {
+            userEmail = line[1].toLowerCase().trim();
+        }
+    }
+
+    private void getUserPermissions(String param, String[] line) {
+        if (param.equals("permissions".toLowerCase().trim())) {
+            String trimLine = line[1].trim();
+            String[] permissionsSplit = trimLine.split(" , ");
+            userPermissions = new ArrayList<>(Arrays.asList(permissionsSplit));
+        }
+    }
+
+    private void getUserAttributes(String param, String[] line) {
+        if (param.equals("attributes".toLowerCase().trim())) {
+            userAttributes = new LinkedHashMap<>();
+            // country: ukraine; city: kharkiv
+            String[] attributesSplit = line[1].trim().split("; ");
+            // country: ukraine
+            String[] countrySplit = attributesSplit[0].trim().split(": ");
+            // city: kharkiv
+            String[] citySplit = attributesSplit[1].trim().split(": ");
+            // country
+            // ukraine
+            userAttributes.put(countrySplit[0], countrySplit[1]);
+            // city
+            // kharkiv
+            userAttributes.put(citySplit[0], citySplit[1]);
+        }
+    }
+
+    private void getUserCreatedData(String param, String[] line) {
+        if (param.equals("createdAt".toLowerCase().trim())) {
+            String[] split = line[1].split("T");
+            String[] splitDate = split[0].split("-");
+            String[] splitTime = split[1].split(":");
+
+            Calendar calendar = Calendar.getInstance();
```

Comment:
I'd recommend you to take a look at [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html).
It much easier to use and less error-prone, comparing to manual string parsing.